### DISCUSSION TITLE
[Feature] Find duplicates - replace deleted duplicate in album

### DISCUSSION THREAD

**Discussion by LokiGT** [immich-app/immich#13687] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [x] Yes

### The feature

Search for duplicates - replacing a deleted duplicate in an album.
When deleting a duplicate of a photo located in an album, the original photo remains without an album.
It is necessary to add the original photo to the same album from which its duplicate was deleted.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10262
